src/EnergyPlus/DirectAirManager.cc

```diff
@@ -625,7 +634,7 @@ namespace DirectAirManager {
 		MaxAirVolFlowRateUser = 0.0;
 		SizingDesRunThisZone = false;
 
-		if ( CurZoneEqNum > 0 ) {
+		if ( ( CurZoneEqNum > 0 ) && ( CurTermUnitSizingNum > 0 ) ) {
```

Comment:
CureZoneEqNum is still used a few lines down for `CheckThisZoneForSizing( CurZoneEqNum, SizingDesRunThisZone );`